src/main/java/org/cactoos/collection/Sorted.java

```diff
@@ -81,16 +80,16 @@ public Sorted(final Comparator<T> cmp, final T... src) {
      */
     public Sorted(final Comparator<T> cmp, final Iterable<T> src) {
         super(
-            new Sticky<>(
-                new CollectionOf<>(
+            new Unchecked<>(
+                new org.cactoos.scalar.Sticky<>(
```

Comment:
@fanifieiev if I'm right, this replacement is also out of scope (`CollectionOf should be deleted`). Maybe it's better to leave a todo?

Reply:
@iakunin Please see [here](https://github.com/yegor256/cactoos/pull/1256#discussion_r356633140)

Reply:
@fanifieiev please see https://github.com/yegor256/cactoos/pull/1256#discussion_r356656070

Reply:
@fanifieiev now (after https://github.com/yegor256/cactoos/pull/1256#discussion_r356657444) this replacement if fine for me, thank you.